<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="preBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modeBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CIO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="listBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PlayList</string>
    </property>
   </widget>
   <widget class="QLabel" name="song_name">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>..</string>
    </property>
   </widget>
   <widget class="QLabel" name="song_author">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>..</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="song_album">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="song_score">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>110</width>
      <height>130</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>130</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>130</height>
     </size>
    </property>
    <property name="text">
     <string>abc</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>79</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="ontopCheck">
       <property name="text">
        <string>on Top</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="muteCheck">
       <property name="text">
        <string>Mute</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_Music"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionStop_Continue"/>
    <addaction name="actionPreview"/>
    <addaction name="actionNext"/>
    <addaction name="separator"/>
    <addaction name="actionVol_Up"/>
    <addaction name="actionVol_Down"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuControl"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Music">
   <property name="text">
    <string>Add Music</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionStop_Continue">
   <property name="text">
    <string>Stop/Continue</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="text">
    <string>Preview</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionVol_Up">
   <property name="text">
    <string>Vol Up</string>
   </property>
  </action>
  <action name="actionVol_Down">
   <property name="text">
    <string>Vol Down</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>preBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>preBtnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopBtnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>nextBtnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>listBtnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>modeBtnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ontopCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTopChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>muteCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>muteChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>preBtnPressed()</slot>
  <slot>stopBtnPressed()</slot>
  <slot>nextBtnPressed()</slot>
  <slot>listBtnPressed()</slot>
  <slot>modeBtnPressed()</slot>
  <slot>onTopChecked(bool)</slot>
  <slot>muteChecked(bool)</slot>
 </slots>
</ui>
